### ISSUE TITLE
Support "aria-keyshortcuts" in react-aria-components

### ISSUE THREAD

**Issue by gregberge** [adobe/react-spectrum#6478]
### Provide a general summary of the feature here

I want to be able to set `aria-keyshortcuts` on a Button.

### 🤔 Expected Behavior?

```tsx
<Button aria-keyshortcuts="S" />
```

It should add the property on the element.

### 😯 Current Behavior

It does not work.

### 🔦 Context

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts

### 🧢 Your Company/Team

Argos

**Comment by reidbarber** [comment]
I think we could add this to `AriaBaseButtonProps` here: https://github.com/adobe/react-spectrum/blob/cef3714bb8816a4ee64ffb1d0730f2669738d2e4/packages/%40react-types/button/src/index.d.ts#L49-L63

**Comment by snowystinger** [comment]
Just to make sure I understand, just putting the attribute on one of our elements won't do anything other than read off to AT? 

It doesn't actually *do* anything right? If so, then I think this is fine. Though it sounds like it's supported on every HTML element? it's going to be harder to decide which components it makes sense on. What all do you want it on so far? Just button?

**Comment by reidbarber** [comment]
@snowystinger Yeah, I think just Button is a good start, and maybe Link as well.
